src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1083,7 +1083,7 @@
   <data name="UnpinFolderFromSidebar" xml:space="preserve">
     <value>Unpin from Sidebar</value>
   </data>
-  <data name="SidebarNetworkDrives" xml:space="preserve">
+  <data name="NetworkDrives" xml:space="preserve">
```

Comment:
Shouldn’t be Network Locations?

Reply:
It seems that it should be Network and NetworkDrives should be no longer used anywhere.

Reply:
I deleted this resource.